packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -6052,6 +6052,7 @@ class Wrap extends MultiChildRenderObjectWidget {
     this.textDirection,
     this.verticalDirection = VerticalDirection.down,
     this.clipBehavior = Clip.none,
+    this.maxLines,
```

Comment:
There should probably be some assertions here that maxLines is a valid value. For example, what happens if someone provides a negative number?